### PR TITLE
Use hosted copy for app

### LINKED ISSUE #2: How will SearchOcean be displayed?
Maybe load the content using <a href="https://isomorphic-git.org/">isomorphic-git</a>?

### PR DESCRIPTION
**Pull request by hello-smile6**
I tested it, and it works!
Fixes #2

### CODE REVIEW

**Comment by hello-smile6** [comment]
@JaydenDev

File: main.js [hunk 1]
```diff
@@ -9,12 +9,12 @@ function createWindow () {
     height: 600,
     webPreferences: {
       preload: path.join(__dirname, 'preload.js'),
-      webViewTag: true
+      webViewTag: false
```
Comment (hello-smile6) [inline main.js:12]: I disabled `<webview>` because it's not needed.

**Comment by JaydenDev** [comment]
idk what the heck you were thinking, it doesn't work. Its still a white screen! so much for this PR lol.

**Comment by JaydenDev** [comment]
nvm forgot to git pull, it works
return to the main searchocean repo

**Comment by hello-smile6** [comment]
> nvm forgot to git pull, it works return to the main searchocean repo

Oh, I'm actually playing https://turbowarp.org/46587498 with a modded client and checking GitHub every few minutes.

**Pull request merged**